generator/explores/explore.py

```diff
@@ -25,6 +25,19 @@ def to_dict(self) -> dict:
 
     def to_lookml(self) -> dict:
         """Generate LookML for this explore."""
+        base_lookml = {}
+        for view_type, view in self.views.items():
+            if "join" in view_type:
```

Comment:
Can you add some comments around here?

Reply:
I will - I think we need to formalize the `view_types` a bit more. The way I'm starting to see it in my head is:
`base_view` - That's the one this explore is created from. Usually the explore name will match this view's name
`extended_view*` - Any views we are extending to create the base view. For example, `client_counts` extends `baseline_clients_last_seen_table`
`join_view*` - Any views we are joining in the explore. This will be for e.g. event funnels